src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const standardPrice = days * pricePerDay;
+
+  if (days >= LONG_TERM) {
+    return standardPrice - 50;
+  } else if (days >= SHORT_TERM) {
+    return standardPrice - 20;
+  }
```

Comment:
no needs to use `else if` when you use early return
```suggestion
  if (days >= LONG_TERM) {
    return standardPrice - 50;
  } 
  
  if (days >= SHORT_TERM) {
    return standardPrice - 20;
  }
```